File: src/main/java/com/procurement/access/model/entity/TenderProcessEntity.java
package com.procurement.access.model.entity;

import java.util.Date;
import java.util.UUID;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class TenderProcessEntity {

    private String cpId;

    private UUID token;

    private String owner;

    private String stage;

    private Date createdDate;

    private String jsonData;
}

File: src/main/kotlin/com/procurement/access/model/entity/TenderProcessEntity.kt
package com.procurement.access.model.entity

import com.procurement.access.domain.model.Cpid
import com.procurement.access.domain.model.Ocid
import java.time.LocalDateTime
import java.util.*


data class TenderProcessEntity(

    var cpId: Cpid,

    var token: UUID,

    var owner: String,

    var ocid: Ocid,

    var createdDate: LocalDateTime,

    var jsonData: String
)
